packages/flutter_tools/lib/src/plugins.dart

```diff
@@ -343,12 +354,20 @@ class Plugin {
     return null;
   }
 
-  static String? _getPluginDartClassForPlatform(YamlMap platformsYaml, String platformKey) {
+  static DartPluginClassAndFilePair? _getPluginDartClassForPlatform(
+    YamlMap platformsYaml,
+    String platformKey,
+    String name,
```

Comment:
`name` is unclear in this context; it needs a more specific variable name.

Also, please used named parameters for the strings now that there is more than one.

Reply:
changed to `pluginName`. `platformKey` and `pluginName` are named parameters now.